eng/bomgenerator/samplereport.txt

```diff
@@ -0,0 +1,21 @@
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - We dropped the following dependencies from the input list.
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-communication-chat:1.0.0, Reason com.azure:azure-core:1.14.1
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-communication-common:1.0.3, Reason com.azure:azure-core-http-netty:1.10.0
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-communication-identity:1.1.1, Reason com.azure:azure-core:1.17.0
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-communication-phonenumbers:1.0.3, Reason com.azure:azure-core:1.17.0
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-communication-sms:1.0.3, Reason com.azure:azure-core:1.17.0
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-cosmos:4.17.0, Reason org.slf4j:slf4j-api:1.7.31
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-digitaltwins-core:1.1.1, Reason com.fasterxml.jackson.core:jackson-annotations:2.12.2
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-messaging-eventgrid:4.4.0, Reason com.azure:azure-core-http-netty:1.10.0
+[main] INFO com.azure.tools.bomgenerator.BomGenerator - Dependency com.azure:azure-messaging-eventhubs-checkpointstore-blob:1.8.1, Reason com.azure:azure-core-http-netty:1.10.0
```

Comment:
>com.azure:azure-core-http-netty: [](http://example.com/codeflow?start=138&length=32)

A very large subset of assemblies are getting dropped due to netty version - I do not see any pending PR for it yet. How have we traditionally solved this.

Do we have tooling that helps migrate these libraries to higher version? Something we need to invest in?

Reply:
Closing this for now. Will use the BOM update CR to talk about this more.